src/modules/RA/DataLayer/components/tabs/PopupTab/PopupFielSelect.js

```diff
@@ -23,17 +23,27 @@ const FieldSelect = ({
         || !popupfields.find(({ sourceFieldId }) => f.sourceFieldId === sourceFieldId))
     : fields), [fields, popupfields, selected]);
 
+  const validateLabel = React.useCallback(v => (v ? undefined : 'datalayer.form.popup.error.required'), []);
```

Comment:
`v` stands for label ?

Reply:
`v` for value, `validate` function receive a value to validate as first argument. But I agree `v` is not clear enough. I will change it

Reply:
It has been removed since I used the `required` validator from react-admin https://marmelab.com/react-admin/CreateEdit.html#per-input-validation-built-in-field-validators